rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -636,13 +636,7 @@
         "type": "datepicker",
         "templateOptions": {
           "hideLabel": true,
-          "wrappers": [
-            "toggle-switch"
-          ],
-          "toggle-switch": {
-            "label": "New acquisition",
-            "description": "If this option is enabled, this item is going to appear in the new acquisition list of the library."
-          }
+          "defaultValueExpression": "new Date().toLocaleDateString('en-CA')"
```

Comment:
I'm not fan about use a "localeDate" parser. I'm feeling more safer using an ISOFormat.
Maybe `new Date().toISOString().split('T')[0]` ?

Reply:
Changed with the new proposal